ChronoKeeper/DataCollectionClient.h

```diff
@@ -0,0 +1,91 @@
+
+#ifndef DATA_COLLECTION_CLIENT_H
+#define DATA_COLLECTION_CLIENT_H
+
+#include <iostream>
+#include <thallium/serialization/stl/string.hpp>
+#include <thallium.hpp>
+
+
+namespace tl = thallium;
+
+namespace chronolog
+{
+
+typedef std::string ChronicleName;
+typedef std::string StoryName;
+typedef uint64_t StoryId;
+
+class DataCollectionClient
```

Comment:
Can we rename this class to something else?  This class is used to exchange internal metadata between Visor and Keeper. It is not on the data path. We call the data structure to store actual incoming events Data Repo. Maybe we can call this DataRepoAdminClient and DataRepoAdminService.